app/grants/models.py

```diff
@@ -118,7 +118,10 @@ def active_clrs_sum(self):
 
 
 class GrantCLR(SuperModel):
-    round_num = models.CharField(max_length=15, help_text="CLR Round Number")
+    
+    customer_name = models.PositiveIntegerField(help_text="CLR Round Number")
```

Comment:
@owocki what is the need for customer name?

Reply:
this is for if we need to do a custom round for zcash, matic, filecoin etc.